.pipelines/release.yml

```diff
@@ -71,6 +71,11 @@ jobs:
       
   - task: NuGetToolInstaller@1
     displayName: Use NuGet Installer latest
+  
+  - task: PowerShell@2
+    displayName: Replace Stubbed Files
```

Comment:
this doesn't seem to (1) replace files or (2) have anything to do with stubs.

I feel like whether `ExperimentationLive` is set to `true` should be entirely handled via an MSBuild condition, like this:

```xml
<ExperimentationLive Condition="'$(ExperimentationLive)' == ''">False</ExperimentationLive>
```

This lets us set it only via msbuild arguments, ala `msbuild blah /p:ExperimentationLive=true`, without patching the project files on disk.

Reply:
another benefit of doing it this say is that we aren't "dirtying" the git repository; some SBOM tools might complain that powertoys was built with a dirty working copy, which could indicate an untracable compromise.

Reply:
Makes sense to me! I'll remove the unstub script and leave it up the msbuild args to change the value of the ExpLive prop.